Ray.jl/src/runtime.jl

```diff
@@ -225,17 +225,14 @@ function task_executor(ray_function, returns_ptr, task_args_ptr, task_name,
 
     # TODO: support multiple return values
     # https://github.com/beacon-biosignals/ray_core_worker_julia_jll.jl/issues/54
-    buffer_data = Vector{UInt8}(sprint(serialize, result))
-    buffer_size = sizeof(buffer_data)
-    buffer = rayjll.LocalMemoryBuffer(buffer_data, buffer_size, true)
-    push!(returns, buffer)
+    push!(returns, _put(result))
```

Comment:
yeah, this to me feels a bit misleading to call it "put", since all it's doing is serializing to a buffer, not actually doing a put...